src/Systems/SpawnerSystem.h

```diff
@@ -3,6 +3,7 @@
 #include "SpawnerComponent.h"
 #include "System.h"
 #include "TransformComponent.h"
+#include "comm.pb.h"
```

Comment:
co to jest

Reply:
Importuję protobufa, żeby móc używać zdefniowanych w nim struktur i wiadomości takich jak np. comm::Enemy albo comm::StateUpdate.